### PR TITLE
vtk: Update 9.1.0 --> 9.2.6

### PR DESCRIPTION
**Pull request by Dave-Allured**
#### Description

* Update to VTK 9.2.6, latest stable (as of 2023 Sept 17).
* Add myself as co-maintainer.
* On request, remove a previous co-maintainer.

###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

CI only.  OS 11_x86, 12_x86, 13_x86.

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] (**N/A**) referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [x] checked your Portfile with `port lint --nitpick`?
- [ ] tried existing tests with `sudo port test`?
- [ ] tried a full install with `sudo port -vst install`?
- [ ] tested basic functionality of all binary files?
- [ ] checked that the Portfile's most important [variants](https://trac.macports.org/wiki/Variants) haven't been broken?

### CODE REVIEW

File: graphics/vtk/Portfile [hunk 1]
```diff
@@ -14,15 +14,18 @@ compiler.cxx_standard 2011
...
+revision            0
 categories          graphics devel
 platforms           darwin
 license             BSD
 
 set branch          [join [lrange [split ${version} .] 0 1] .]
 
-maintainers         {stromnov @stromnov} {@rdbisme gmail.com:rubendibattista} openmaintainer
+maintainers         {stromnov @stromnov} \
+                    {@rdbisme gmail.com:rubendibattista} \
```
Comment (rdbisme) [inline graphics/vtk/Portfile:26]:
```suggestion
```

Comment (rdbisme) [reply graphics/vtk/Portfile:26]: Let's remove me since I don't have access to the hardware anymore.

**Review by rdbisme — CHANGES_REQUESTED** [review]

File: graphics/vtk/Portfile [hunk 2]
```diff
@@ -100,7 +102,7 @@ variant qt5 description {Add Qt5 support.} {
 }
 
 # Supported pythons
-set python_versions {27 36 37 38 39 310 311}
+set python_versions {36 37 38 39 310 311}
```
Comment (reneeotten) [inline graphics/vtk/Portfile:105]: did you verify that the PY27 subport doesn't have dependents anymore (of that applies to the PY36 subport as well, feel free to  remove that one as well).

Comment (Dave-Allured) [reply graphics/vtk/Portfile:105]: > did you verify that the PY27 subport doesn't have dependents anymore

No, that did not occur to me.  I will restore PY27.

Comment (reneeotten) [reply graphics/vtk/Portfile]: You should check whether that's the case and if not, by all means remove the PY27 subport

Comment (Dave-Allured) [reply graphics/vtk/Portfile]: CI has now accepted the PY27 subport on default builds.  Is it okay with you if I leave in the PY27 subport?  If so, please mark this review thread as resolved.  Thanks.

Tasks remaining:  Squash commits; re-lint; clean up commit message.

File: graphics/vtk/Portfile [hunk 3]
```diff
@@ -14,15 +14,17 @@ compiler.cxx_standard 2011
...
+revision            0
 categories          graphics devel
 platforms           darwin
 license             BSD
 
 set branch          [join [lrange [split ${version} .] 0 1] .]
 
-maintainers         {stromnov @stromnov} {@rdbisme gmail.com:rubendibattista} openmaintainer
+maintainers         {stromnov @stromnov} \
+                    Dave-Allured \
```
Comment (pmetzger) [inline graphics/vtk/Portfile:26]: This is not a correct maintainers line.

File: graphics/vtk/Portfile [hunk 4]
```diff
@@ -14,15 +14,17 @@ compiler.cxx_standard 2011
...
+revision            0
 categories          graphics devel
 platforms           darwin
 license             BSD
 
 set branch          [join [lrange [split ${version} .] 0 1] .]
 
-maintainers         {stromnov @stromnov} {@rdbisme gmail.com:rubendibattista} openmaintainer
+maintainers         {stromnov @stromnov} \
+                    @Dave-Allured \
```
Comment (pmetzger) [inline graphics/vtk/Portfile:26]: Generally, it's expected that you'll provide a (obfuscated) email address so you can be contacted if there's trouble with your port. See the maintainer entry you are removing.

**Comment by reneeotten** [comment]
there seems to be a stray Portfile now as well. Secondly, you should "rebase" instead of merging branches in here as they'll result in a merge commit, which we don't want.

**Comment by Dave-Allured** [comment]
Fumbled, uploaded Portfile to wrong place.  Retrying...

**Comment by Dave-Allured** [comment]
Fixed now.  The last commit includes everything.  Can you merge only the last commit, and avoid the "merge commit"?  Do you see any other problems?

**Comment by reneeotten** [comment]
> Fixed now.  The last commit includes everything.  Can you merge only the last commit, and avoid the "merge commit"?  Do you see any other problems?

as I said before you should "rebase" with master and then force-push to this branch.

**Comment by Dave-Allured** [comment]
I am trying to avoid using git on corporate machines, for security.  Therefore I am trying to create all commits and PR's directly on github website.  Is it possible to force-push, amend commit, or squash commits using only github website?  Is there another way?

**Comment by Dave-Allured** [comment]
Closing now because this PR is too messed up.  Will start another PR later.

**Pull request closed without merging**